src/code/modules/parent-gametes.js

```diff
@@ -0,0 +1,134 @@
+
+/*
+  state contains:
+    pools: array of two arrays, 0: father's gametes, 1: mother's gametes
+    selectedIndices: array of two indices representing the index of
+                      the selected gamete (if any) for each parent
+ */
+const initialState = Immutable({ pools: Immutable([Immutable([]), Immutable([])]),
+                                  selectedIndices: Immutable([null, null]) });
+
+/*
+ * reducer
+ */
+export default function reducer(state = initialState, action) {
+  let newState = {}, pools, selectedIndices;
+  switch (action.type) {
+    case GAMETE_POOLS_RESET:
+      return initialState;
+    case GAMETES_ADDED_TO_POOL:
+      pools = state.pools.map((pool, index) => {
+                  return index === action.sex
+                          ? pool.concat(action.gametes)
+                          : pool;
+                });
+      return state.merge({ pools });
+    case GAMETE_SELECTED:
+      selectedIndices = state.selectedIndices.map((selectedIndex, index) => {
+                                        return (index === action.sex) ? action.index : selectedIndex;
+                                      });
+      return state.merge({ selectedIndices });
+    case actionTypes.OFFSPRING_KEPT:
+      // remove selected gamete from pool after offspring drake is kept
+      newState.pools = state.pools.map((pool, poolIndex) => {
+                          return pool.map((gamete, gameteIndex) => {
+                            return gameteIndex === state.selectedIndices[poolIndex]
+                                    ? null : gamete;
+                          });
+                        });
+      // fallthrough intentional
+    case actionTypes.GAMETES_RESET:
+      newState.selectedIndices = Immutable([null, null]);
+      return state.merge(newState);
+    default:
+      return state;
+  }
+}
+
+/*
+ * selectors
+ */
+export const fatherGametePool = state => state.pools[0];
+export const motherGametePool = state => state.pools[1];
+export const gametePoolSelector = sex => sex ? motherGametePool : fatherGametePool;
+export const fatherSelectedGameteIndex = state => state.selectedIndices[0];
+export const motherSelectedGameteIndex = state => state.selectedIndices[1];
+export const fatherSelectedGamete = createSelector(
+                                      fatherGametePool,
+                                      fatherSelectedGameteIndex,
+                                      (pool, index) => pool[index]);
+export const motherSelectedGamete = createSelector(
+                                      motherGametePool,
+                                      motherSelectedGameteIndex,
+                                      (pool, index) => pool[index]);
+
+/*
+ * action creators
+ */
+export function addGametesToPool(sex, gametes) {
+  return {
+    type: GAMETES_ADDED_TO_POOL,
+    sex,
+    gametes: cloneDeep(gametes),
+    meta: {
+      logTemplateState: true,
```

Comment:
I was just following the pattern set out by whatever other action I copied initially. I guess I'll set it to false for now and we can figure out what logState should do at a later date.